lib/jekyll-jupyter-notebook/converter.rb

```diff
@@ -54,6 +54,7 @@ def convert_notebook(content, config)
           "--stdout",
         ]
         command_line << "--no-prompt" unless config.fetch("prompt", true)
+        command_line << "--no-input" unless config.fetch("input", true)
```

Comment:
Could you sort in alphabetical order?

```suggestion
        command_line << "--no-input" unless config.fetch("input", true)
        command_line << "--no-prompt" unless config.fetch("prompt", true)
```